README.md

```diff
@@ -0,0 +1,12 @@
+Codex Research
+===============
+
+Contains research for the Codex peer-to-peer storage network. Top-level structure:
+
+* analysis - simulations, notebooks, etc;
```

Comment:
we can make this links, for easier navigation ;-)

Reply:
Good point, will add :-)